saleor/events/bases.py

```diff
@@ -0,0 +1,36 @@
+from django.core.exceptions import ImproperlyConfigured
+from django.db import models
+
+
+class EventManager:
```

Comment:
I really appreciate the design but to be honest, the whole thing seems over-engineered. The whole "events" design could be implemented as a single model class (in our case `OrderEvent`) plus a bunch of module functions to create specific events. What we have now is a manager object which the only purpose is to nicely create multiple objects at once. I still think that this is not worth the effort as we never create that many of them at once.